modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/operation/ComponentMessageProcessor.java

```diff
@@ -666,8 +861,9 @@ protected ParameterValueResolver getParameterValueResolver() {
     }
   }
 
-  @Override
-  public abstract ProcessingType getProcessingType();
+  public ProcessingType getInnerProcessingType() {
+    return CPU_LITE;
```

Comment:
add a clarifying comment about the inner chain processing type being the one that actually reflects the operation